www/src/components/markdown-page-footer.js

```diff
@@ -14,16 +14,38 @@ export default class MarkdownPageFooter extends React.Component {
       <>
         <hr sx={{ display: `none` }} />
         {this.props.page && (
-          <a
-            sx={{ variant: `links.muted`, mt: 9 }}
-            href={`https://github.com/gatsbyjs/gatsby/blob/master/${
-              this.props.packagePage ? `packages` : `docs`
-            }/${this.props.page ? this.props.page.parent.relativePath : ``}`}
+          <div
+            sx={{
+              display: `flex`,
+              alignItems: `center`,
+              justifyContent: `space-between`,
+              mt: 9,
+            }}
           >
-            <EditIcon sx={{ marginRight: 2 }} />
-            {` `}
-            Edit this page on GitHub
-          </a>
+            <a
+              sx={{ variant: `links.muted` }}
+              href={`https://github.com/gatsbyjs/gatsby/blob/master/${
+                this.props.packagePage ? `packages` : `docs`
+              }/${this.props.page ? this.props.page.parent.relativePath : ``}`}
+            >
+              <EditIcon sx={{ marginRight: 2 }} />
+              {` `}
+              Edit this page on GitHub
+            </a>
+            {this.props.page &&
+              this.props.page.parent &&
+              this.props.page.parent.fields &&
+              this.props.page.parent.fields.gitLogLatestDate && (
```

Comment:
NIT: could we use optional chaining here?

```diff
-            {this.props.page &&
-              this.props.page.parent &&
-              this.props.page.parent.fields &&
-              this.props.page.parent.fields.gitLogLatestDate && (
+            {this.props.page?.parent?.fields?. gitLogLatestDate && (
```

Reply:
@kraynel I think after this the PR should be good to go 👍

Reply:
Updated and rebased!